src/config.rs

```diff
@@ -118,6 +120,15 @@ fn try_read_config() -> Result<Config, String> {
         Err(_) => false,
     };
 
+    let gpu_device_id = match env::var("SMELTER_GPU_DEVICE_ID") {
+        Ok(id) => Some(
+            id.parse::<u32>()
+                .map_err(|_| "SMELTER_GPU_DEVICE_ID has to be a valid number")?,
+        ),
+        Err(_) => None,
+    };
+    let gpu_driver_name = env::var("SMELTER_GPU_DRIVER_NAME").ok();
```

Comment:
```suggestion
    let gpu_driver_name = env::var("SMELTER_GPU_DEVICE_DRIVER").ok();
```
I would also prefer using consistent match if clipy allows it.

When adding this to docs, show example output and examples of what values could be provided here